plugins/modules/ini_file.py

```diff
@@ -198,7 +227,7 @@ def match_opt(option, line):
 
 def match_active_opt(option, line):
     option = re.escape(option)
-    return re.match('()( |\t)*(%s)( |\t)*(=|$)( |\t)*(.*)' % option, line)
+    return re.match('( |\t)*(%s)( |\t)*(=|$)( |\t)*(.*)' % option, line)
```

Comment:
It looks like you are removing a previous PR. Please undo that.

Reply:
Oh I think I applied an old version of the file somehow. I'll start over and make a new pr.